### PR TITLE
Adding "audioMain" to Nuke reviews.

### PR DESCRIPTION
**Pull request by tokejepsen**
- also added get_latest_version method to library.

Resolves https://github.com/pypeclub/pype/issues/368

### CODE REVIEW

File: pype/plugins/nuke/publish/collect_review.py [hunk 1]
```diff
@@ -23,6 +26,21 @@ def process(self, instance):
         if not node["review"].value():
             return
 
+        # Add audio to instance if it exists.
+        try:
+            version = pype.api.get_latest_version(
+                instance.context.data["assetEntity"]["name"], "audioMain"
```
Comment (mkolar) [inline pype/plugins/nuke/publish/collect_review.py:32]: asset name should be a class attribute to allow overriding it in the presets. Different studios might have different convention on naming here.

Comment (tokejepsen) [reply pype/plugins/nuke/publish/collect_review.py:32]: You mean `audioMain`?

Comment (mkolar) [reply pype/plugins/nuke/publish/collect_review.py:32]: yes. `audioMain`. it's entirely possible that a studio would want to bring in `audioVoices` or another subset.

**Comment by mkolar** [comment]
I'll merge this now, and do the customisation in a new PR

**Pull request merged**